src/api/auth.js

```diff
@@ -0,0 +1,19 @@
+import API from './axios'
+
+export const loginUser = async (data) => {
+   let response = {}
+   await API.post('/auth/local', {
+      ...data,
+   })
+      .then((res) => {
+         response = {
+            ok: true,
+            jwt: res.data.jwt,
+            user: res.data.user,
+         }
+      })
+      .catch((e) => {
+         response = { ok: false, error: e.response.data.error.message }
+      })
+   return response
+}
```

Comment:
1. No need to spread `{ ...data }`, you can just pass `data`
2. Avoid mixing async/await and then/catch, use just async await 😄 
3. Whenever you have await, make sure to wrap it in try / catch, otherwise if the promise throws an error, it will be uncaught. If you don't want to process the exceptions on this layer (usually you don't want to handle them here, but in the place that calls this function) - you can just return the promise without awaiting it
4. A better practice is to let the promise throw an exception instead of returning `{ ok: false, ... }`

```suggestion
export const loginUser = async (data) => {
  try {
     const response = await API.post('/auth/local', data);
     const {jwt, user} = response.data;
     return {jwt, user};
  } catch (ex) {
     throw Error(e?.response?.data?.error?.message ?? 'Unknown error');
  }
}
```